src/main/java/net/glowstone/block/blocktype/BlockRedstoneTorch.java

```diff
@@ -55,7 +55,7 @@ public void placeBlock(GlowPlayer player, GlowBlockState state, BlockFace face,
     @Override
     public void updatePhysics(GlowBlock me) {
```

Comment:
@mastercoms Can you test it with the latest commit? I have yet to encounter the bug with that change